src/types/leafygreen.ts

```diff
@@ -1,26 +0,0 @@
-import Card from "@leafygreen-ui/card";
```

Comment:
(placement of comment is random) Does `BaseToggle/index.tsx` need `DisclaimerProps`?

Reply:
I mentioned [here](https://github.com/evergreen-ci/spruce/pull/1852#discussion_r1211764099) that I think it's sufficient to only add types when an error is thrown since that's we've done up until this point. These components don't throw errors, I think because they do not use LG's Polymorphic under the hood. Same applies below!